website/blog/2020-06-30-data-lake-analytics-blog.md

```diff
@@ -0,0 +1,127 @@
+c. JDBC/ODBC support<br>
+d. Compatible with BI tools<br>
+3. Heterogeneous Data Source<br>
+a. Enable OSS Data analysis requirements<br>
+b. Enable Table Store Data for SQL querying service<br>
+c. Federated data analysis across multiple database instances<br>
+d. Ease of interconnection analysis for multiple data sources<br>
+e. Any forms of analytics in “Data Lake”<br>
+4. Optimized Computing Engine<br>
+a. Support for unstructured data<br>
+b. Vectorized execution and optimization<br>
+c. Operator pipeline optimization<br>
+d. Resource isolation and priority<br>
+
+
+
+## DLA Use Cases
+
+
+### Typical Data Flow Scenarios:
+
+Generally DLA empowers cloud users analytic capability in 3 typical user data flow scenarios:<br>
+1. The cloud user uploads the business generated data such as Log, CSV, JSON and other files, directly to OSS(Alicloud Object Storage Service), and then uses DLA to directly point to the file or folder for which the table is created to query. Then the user may use BI tools to analyze and visualize business data analytics.<br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case1.png)
+<br><br>
+2. The cloud user can directly copy and upload data in the Hadoop’s legacy format, such as Parquet, ORC, RCFile, Avro, and other formats, to OSS, then use DLA to directly point files or folders for which the table is created to query. Then the user may use BI tools for business data analysis and visualization. <br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case2.png)
+<br><br>
+3. In order to provide better query performance and low storage cost for subsequent data analysis on OSS, data can be converted into Parquet or ORC format to improve the cost performance of repeated data analysis.<br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case3.png)
+<br><br>
+
+### Alibaba Use Case Studies: 
+
+#### Customer Case #1: Log Data Analytics
+
+Cloud customers generate log data which is stored on OSS. The challenge is that those log data are unstructured and typically they are in Petabytes of data size. <br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case1_logAnalysis.png)
+<br><br>
+So in order to make it available to analyze, DLA’s massive parallel processing engine is the key to business success. It allows users to do full-volume, personalized log data analysis by using DLA’s Hive connector. Additionally log data can be synchronized to OSS with a max delay of 5 minutes with Alibaba Cloud SLS that can feed data to DLA in near real-time.
+<br><br>
+
+#### Customer Case #2: Genetic Data Analytics
+
+
+One of Alicloud’s data vendors is in the Genetic engineering industry. The data volume is gigantic and the data are stored in various data sources. Its raw data containing all bioinformatics for storing gene sequence variations are stored as Variant Call Format(VCF). The challenge is that the Genetic datasets are huge and need to be analyzed promptly. 
```

Comment:
Maybe

"The data volume is gigantic and the data is stored in various data sources. This is raw bioinformatics data storing gene sequence variations in the Variant Call Format (VCF). The challenge is that genetic datasets are huge, but still need to be analyzed promptly."

Reply:
fixed, thanks